lib-find-meal/proguard-rules.pro

```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
```

Comment:
Don't ask me what this file is for because I'm not experienced enough to know yet but I just leave them as-is for now!